STATUS.md

```diff
@@ -0,0 +1,487 @@
+# DCS Functional Parity
+
+The DCS gRPC Server API implementation is still a work in progress, and therefore the following API functions have
+been implemented.
+
+The unimplemented functions is not committed to a roadmap. We do envision full DCS API equivalence at some point in the
+future; pull requests are welcomed for expanding the API equivalency.
+
+---
+
+## Singletons
+
+### Atmosphere Singleton
+- [x] `getWind`
+- [x] `getWindWithTurbulence`
+- [x] `getTemperatureAndPressure`
+
+### Coalitions Singleton
+- [ ] `addGroup`
+  - [ ] Sea
+    - [ ] Group
+    - [ ] Units
+    - [ ] Waypoints
+    - [ ] Tasks
+  - [ ] Ground
+    - [ ] Group
+    - [ ] Units
```

Comment:
Ground group and Units have been done (At least to the point that a NASAM site can be spawned)